.circleci/config.yml

```diff
@@ -3498,6 +3498,39 @@ workflows:
           requires:
             - check-if-nightly-tests-should-run
 
+  # ATTENTION: When adding jobs to the "nightly-quarantine" workflow, make sure you have a
+  # "requires" that lists "check-if-nightly-tests-should-run", so that the job
+  # only runs if "check-if-nightly-tests-should-run" allows it to.
+  nightly-quarantine:
+    jobs:
+      - check-if-nightly-tests-should-run:
+          context:
+            # Provides the GITHUB_USERNAME and GITHUB_TOKEN enviroment variable
+            # that's required by the "gh" command for authentication.
+            - github-write
+      - test-e2e-aws:
+          name: test-e2e-gpu-nightly-quarantine
+          context: aws
+          matrix:
+            parameters:
+              parallelism: [2]
```

Comment:
maybe parallelism of 1 is enough? there should not be a ton of tests there

Reply:
Sounds good to me.